gax/test/unit/regapic.ts

```diff
@@ -202,7 +207,10 @@ describe('REGAPIC', () => {
 
       gaxGrpc.createStub(libraryService, stubOptions).then(libStub => {
         libStub.getShelf(requestObject, {}, {}, (err?: {}, result?: {}) => {
-          assert.strictEqual(spy.getCall(0).returnValue?.queryString, '');
+          assert.doesNotMatch(
```

Comment:
Not sure what this is testing (what is $alt)? If possible, could we search for what we are looking for? This will also allow readability in the code (it will help future maintainers understand what the behavior should look like). Or, if we also want to test it does not contain $alt, let's also add a positive-looking test. It's also a bit confusing to test the first call OR an empty string?

Reply:
Good call. I reverted these changes. They were artifacts from a different implementation, but I left them in because I thought they made the tests more robust. However, with your comment I see the downsides.